Tests/MockContentsquareInstance.swift

```diff
@@ -11,45 +11,57 @@ import XCTest
 
 class MockContentsquareInstance: ContentsquareCommand {
 
-    var sendScreenViewCallCount = 0
-    var sendTransactionCallCount = 0
-    var sendDynamicVarCallCount = 0
-    var stopTrackingCallCount = 0
-    var resumeTrackingCallCount = 0
-    var forgetMeCallCount = 0
-    var optInCallCount = 0
-    var optOutCallCount = 0
+    // Tracking last parameters
+    var lastScreenName: String?
+    var lastTransactionInfo: (price: Double, currency: String, transactionId: String?)?
+    var lastDynamicVar: [String: Any]?
+    var lastUserId: String?
+    var lastCustomVarsInfo: (screenName: String, customVars: [[String: Any]])?
+    
+    // Tracking calls
+    var didStopTracking = false
+    var didResumeTracking = false
+    var didForgetMe = false
+    var didOptIn = false
+    var didOptOut = false
```

Comment:
This is fine here, but in general is not a necessary change to make. 
The counter helps to test that the method is not called more than once.

Reply:
no in general yes, this specific one is not needed though.